PlayTools/PlayCover.swift

```diff
@@ -10,6 +10,8 @@ public class PlayCover: NSObject {
 
     static let shared = PlayCover()
     var menuController: MenuController?
+    private static var windowInitRetryCount = 0
```

Comment:
Are `windowInitRetryCount` and `maxRetries` used?

Reply:
It's safe to be removed on this PR, but will be added again in another pr